packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -33,6 +38,11 @@ export const theme = DesignToken.create<Theme>({
  * @internal
  */
 export class ThemeProvider extends FoundationElement {
+    @attr({
+        attribute: 'lang'
+    })
+    public override lang: string = document.documentElement.lang;
```

Comment:
I think it would be good to have some unit tests for the new attribute.

Reply:
Added a couple tests to validate connection between attribute and design token.